loadtest/config.go

```diff
@@ -95,14 +95,31 @@ func GetConfig() (*Config, error) {
 	return cfg, nil
 }
 
-// IsValid checks whether a config is valid or not.
+// IsValid reports whether a config is valid or not.
 func (c *Config) IsValid() (bool, error) {
 	if c.ConnectionConfiguration.ServerURL == "" {
 		return false, fmt.Errorf("ServerURL is not present in config")
 	}
 	if c.ConnectionConfiguration.WebSocketURL == "" {
 		return false, fmt.Errorf("WebSocketURL is not present in config")
 	}
+	if c.ConnectionConfiguration.AdminEmail == "" {
```

Comment:
Should we also add checks for the missing fields like `DriverName` and `DataSource` for completeness sake?

Reply:
Actually those are not used at all I think. Maybe we can even remove them from the config.

Reply:
I think originally they were needed to do some sort of bulkloading which then I tried to avoid as much as possible and instead rely on a single system admin to create the initial state.

There might still be some usefulness to have them for local testing, like creating the initial admin account without using the CLI or resetting the db when needed.